projects/orquestra-sdk/pyproject.toml

```diff
@@ -26,8 +26,8 @@ requires-python = ">= 3.8"
 # version we can easily install and test, even if there are older versions with
 # the same API.
 dependencies = [
-    "orquestra-workflow-runtime==0.0.1",
-    "orquestra-workflow-shared==0.0.1",
+    "orquestra-workflow-runtime==0.0.2",
+    "orquestra-workflow-shared==0.0.2",
```

Comment:
**nitpick**: when making a regular release (and not a hotfix), can we bump minor and not patch, please?